tests/test_base_transformers/test_transform_xy_mixin.py

```diff
@@ -0,0 +1,22 @@
+import numpy as np
+import pandas as pd
+
+from feature_engine._base_transformers.mixins import TransformXyMixin
+
+
+class MockTransformer(TransformXyMixin):
+    def transform(self, X):
+        return X.iloc[1:-1].copy()
+
+
+def test_transform_x_y_method(df_vartypes):
+    y = pd.Series(pd.Series(0, index=np.arange(len(df_vartypes))))
+    transformer = MockTransformer()
+    Xt, yt = transformer.transform_x_y(df_vartypes, y)
+    print(Xt)
```

Comment:
I suppose the print was just for debugging :-)

Reply:
yes, thank you!